gotest_local_me/BBQ_test.go

```diff
@@ -0,0 +1,120 @@
+package ginkgo_me
+
+import (
+	mock_beeffarm "github.com/artiomgiza/test_me/pkgs/beef-farm/mock"
+	mock_chickenfarm "github.com/artiomgiza/test_me/pkgs/chicken-farm/mock"
+	mock_coalfarm "github.com/artiomgiza/test_me/pkgs/coal-farm/mock"
+	mock_mangalstore "github.com/artiomgiza/test_me/pkgs/mangal-store/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_coolPriceCalculator_CalculatePrice(t *testing.T) {
+
+	t.Run("calc", func(t *testing.T) {
+		t.Run("when people counter is exceeded max value", func(t *testing.T) {
+			ctrl := gomock.NewController(t)
+			defer ctrl.Finish()
+
+			b := coolPriceCalculator{}
+			_, err := b.CalculatePrice(11)
+			assert.NotNil(t, err)
+			assert.Contains(t, err.Error(),"too much people", "should return error about people exceeded max value")
+		})
+		t.Run("when people counter is not exceeded max value",func(t *testing.T) {
+			t.Run("when get entrecote returns error", func(t *testing.T) {
+				ctrl := gomock.NewController(t)
+				defer ctrl.Finish()
+				peopleCounter := 10
+				mockBeefFarm := mock_beeffarm.NewMockProvider(ctrl)
+				mockBeefFarm.EXPECT().GetEntrecote(peopleCounter).Return(0, errors.New("test_err_7483")).Times(1)
+				t.Run("when get chicken returns error", func(t *testing.T) {
+					mockChickenFarm := mock_chickenfarm.NewMockProvider(ctrl)
+					b := coolPriceCalculator{
+						beefFarm:    mockBeefFarm,
+						chickenFarm: mockChickenFarm,
+					}
+					mockChickenFarm.EXPECT().GetPullet(peopleCounter).Return(0, errors.New("test_err_2393"))
+					_, err := b.CalculatePrice(peopleCounter)
+					assert.NotNil(t, err)
+				})
+				t.Run("when get chicken returns no error", func(t *testing.T) {
+					t.Run("when mangal return error", func(t *testing.T) {
+						// In gincko test this text
+						// this case is not tested
+						// flow is the same as next "when get entrecote returns no error"
+						// but I think we need test this case too. Because we must be sure that meetPrice was overwritten
+					})
+				})
+			})
+			t.Run("when get entrecote returns no error", func(t *testing.T) {
+				ctrl := gomock.NewController(t)
+				defer ctrl.Finish()
+				peopleCounter := 10
+				meetPrice := 100
+				mockBeefFarm := mock_beeffarm.NewMockProvider(ctrl)
+				mockBeefFarm.EXPECT().GetEntrecote(peopleCounter).Return(meetPrice, nil).AnyTimes()
```

Comment:
why do we need `.AnyTimes()` here (and other places)?
In general it's not good, because it will be ok with 0 runs, 1 run, ..., 100500 runs...
I think test must ensure exact number of calls to other pkgs (logic might left the same, but this might point to performance bug)